### PR TITLE
Fix hint text for available points (#3377)

### PR DESCRIPTION
**Pull request by bowmnath**

### CODE REVIEW

File: pages/partials/examQuestionAvailablePoints.ejs [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <% } %>
 
     <!-- Create popover with info about what will happen on next submission -->
-    <button type="button" class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('examQuestionAvailablePointsInfo', {bestScore: best, pointsAvailable: points_list[0], triesRemaining: points_list.length}) %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
+    <button type="button" class="btn btn-xs" data-toggle="popover" data-html="true" data-content="<%= include('examQuestionAvailablePointsInfo', {bestScore: best, pointsAvailable: points_list[0], currentWeight: currentWeight, triesRemaining: points_list.length}) %>" data-placement="auto" data-trigger="focus" data-container="body" tabindex="0">
```
Comment (bowmnath) [inline pages/partials/examQuestionAvailablePoints.ejs:22]: I wasn't sure whether this change was strictly necessary?

File: pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs [hunk 2]
```diff
@@ -206,7 +206,7 @@
 
               <td class="text-center"><%- include('../partials/examQuestionScore', {instance_question}); %></td>
               <% if (assessment.allow_real_time_grading) { %>
-                <td class="text-center"><%- include('../partials/examQuestionAvailablePoints', {open: assessment_instance.open && instance_question.open, points_list: instance_question.points_list}); %></td>
+                <td class="text-center"><%- include('../partials/examQuestionAvailablePoints', {open: assessment_instance.open && instance_question.open, currentWeight: instance_question.points_list_original[instance_question.number_attempts], points_list: instance_question.points_list, highest_submission_score: instance_question.highest_submission_score}); %></td>
```
Comment (bowmnath) [inline pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs:209]: This one had two changes -- I think there was a mistake in the hint text for scores > 0 on the overview page.  They were defaulting to `bestScore == 0`.

File: pages/partials/examQuestionAvailablePoints.ejs (hunk 1 above)
Comment (mwest1066) [reply pages/partials/examQuestionAvailablePoints.ejs:22]: I believe that it's not necessary, but it doesn't really hurt either.

**Review by jonatanschroeder — APPROVED** [review]

File: pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs (hunk 2 above)
Comment (mwest1066) [reply pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs:209]: Good catch!

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**